packages/loot-core/src/server/budget/app.ts

```
@@ -153,6 +165,390 @@
+            }
+          } else {
+            // Manual projection - use manual values if available, otherwise template
+            if (category.is_income) {
+              // Handle income categories
+              if (
+                manualIncomeValues &&
+                manualIncomeValues[period] &&
+                manualIncomeValues[period][category.id] !== undefined
+              ) {
+                projectedAmount = manualIncomeValues[period][category.id];
+              } else if (template && template[category.id]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)